### ISSUE TITLE
Replace 'songs by singer' with 'singer songs' in help.py

### ISSUE THREAD

**Issue by Mohit17067** [swapagarwal/JARVIS-on-Messenger#368]
When we type `help`, Jarvis shows that it can proceed with 'songs by \<singer\>' ( `songs by linkin park` ). But instead, it should show '\<singer\> songs' ( `linkin park songs` ) because I think `music.py` is implemented like this. Changing `help.py` may do the work. Shall I proceed?

**Comment by skakim** [comment]
I agree, "songs by \<singer\>" doesn't work at all. Moreover, for me, "\<singer\> songs" don't work for all bands. The following worked:
- `linkin park songs`
- `metallica songs` 
- `sepultura songs`
- `nightwish songs`
- `pink floyd songs`
- `red hot chili peppers songs`
- `coldplay songs`

The following didn't work (and I didn't receive any response) (maybe **newer bands/singers**?):
- `avicii songs` 
- `ariana grande songs`
- `katy perry songs`
- `madonna songs`
- `justin bieber songs`
- `drake songs`
- `lady gaga songs`
- `eminem songs`
- `miley cyrus songs`
- `shakira songs`

In some cases (rarely), I receive a `I'm sorry; I'm not sure I understand what you're trying to say.` answer:
- `rihanna songs`
- `guns n'roses songs`
- `imagine dragons songs`

**Comment by Mohit17067** [comment]
That must be because the API does not have those singers on their list.
